test/bids-asks/actions/load-bids-asks-test.js

```diff
@@ -24,8 +39,8 @@ describe(`modules/bids-asks/actions/load-bids-asks.js`, () => {
     params: {
       marketID: undefined
     },
-    mock: {
-      state: { marketsData }
+    marketData: {
```

Comment:
Should this be `marketsData` plural?  A bit confusing otherwise.

Reply:
it is a bit confusing:
The default data structure is marketsData (plural)

mockLoadMarketsInfo.loadMarketsInfo = sinon.stub().callsFake((p, cb) => (dispatch, getState) => {
      cb(null, t.marketData)
})

the mock returns the test property marketData as t.marketData